RIOT-OS/RIOT
# make/emulate: use unix sockets with qemu

### Contribution description

make qemu use unixsockets in a temporary directory

### Testing procedure

```
EMULATE=1 BOARD=microbit make all term
```
and
```
EMULATE=1 BOARD=microbit make all debug
```
should work

### Issues/PRs references

might solve #17555
